src/components/checkboxes/index.md.njk

```diff
@@ -65,19 +65,25 @@ You can add hints to checkbox items to provide additional information about the
 
 {{ example({group: "components", item: "checkboxes", example: "hint", html: true, nunjucks: true, open: false, size: "s"}) }}
 
-### Conditionally revealing content
+### Conditionally revealing a related question
 
-You can conditionally reveal content when the user selects a particular checkbox, so they only see content when it’s relevant to them.
+You can ask the user a related question when they select a particular checkbox, so they only see the question when it's relevant to them.
 
-For example, you could reveal a phone number input when the user selects the 'Contact me by phone' option.
+This might make two related questions easier to answer by grouping them on the same page. For example, you could reveal a phone number input when the user selects the 'Contact me by phone' option.
```

Comment:
```suggestion
This might make 2 related questions easier to answer by grouping them on the same page. For example, you could reveal a phone number input when the user selects the 'Contact me by phone' option.
```
Style guide says to write out numbers 2-9 except in expressions like "one or two of them." Feels a mite odd, but so sayeth the scripture...